versions.txt

```diff
@@ -29,6 +29,9 @@ autoinstrumentation-python=0.36b0
 # Should match autoinstrumentation/dotnet/version.txt
 autoinstrumentation-dotnet=0.6.0
 
+# Represents the current release of Golang instrumentation.
+autoinstrumentation-golang=v0.6.4
```

Comment:
The image naming pattern we use for everything omits the v here, not sure it matters / will change in the future, just a thought that we may want to match it here and then add the v in the sprintf in the operator?

Reply:
I can do that if you want.  I chose this value initially as it represents the exact tag value of the image, which is what version values in the file represent.  This keeps the code in main.go consistent.